app/controllers/accounts_controller.rb

```diff
@@ -15,6 +15,7 @@ def show
       sp_name: decorated_session.sp_name,
       decorated_user: current_user.decorate,
       locked_for_session: pii_locked_for_session?(current_user),
+      account_reset_requested: current_user&.account_reset_request.present?,
```

Comment:
My comment on filtering the `account_reset_request` based on the attribute `cancelled_at` applies here.

Reply:
To clarify and consolidate the comments, I believe you still want to filter the `AccountResetRequests` by the property `cancelled_at`.

Apparently you want to show a message to the user that their account deletion request has been cancelled, so you want to search for a `AccountResetRequest` for the current user where the value of `cancelled_at` is within the past 24 hours. The time constraint is checked so that the message will only be displayed for recent cancellations.

Reply:
There is now a [ticket for the banner appearing after the 24 hour waiting period](https://cm-jira.usa.gov/browse/LG-6788) so I believe the work, as it stands meets the requirement.

Reply:
My concern is is with old `AccountResetRequests`. Are they filtered out of the system eventually? If not, it appears necessary to have some code like:

```
  def account_reset_requested?
    AccountResetRequest.where(
      user: current_user,
    ).where('cancelled_at > ?',
            IdentityConfig.store.account_reset_wait_period_days.days.ago).present?
  end
```

Perhaps I'm misunderstanding some of the requirements.

Reply:
This is a solution that works.